codewof/tests/codewof_test_data_generator.py

```diff
@@ -62,7 +64,7 @@ def generate_questions():
 
 
 def generate_users(user):
-    """Generate users for codeWOF tests. Creates two basic users for unit tests."""
+    """Generate users for codeWOF tests. Creates three basic users for unit tests."""
```

Comment:
Since the number is now four, probably best to change it to 'multiple'.
```suggestion
    """Generate users for codeWOF tests. Creates multiple basic users for unit tests."""
```

Reply:
Done